hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -155,6 +155,13 @@ public class HoodieCompactionConfig extends HoodieConfig {
       .withDocumentation("Used by org.apache.hudi.io.compact.strategy.DayBasedCompactionStrategy to denote the number of "
           + "latest partitions to compact during a compaction run.");
 
+  public static final ConfigProperty<String> COMPACTION_SPECIFY_PARTITION_PATH_REGEX = ConfigProperty
+      .key("hoodie.compaction.partition.path.regex")
+      .defaultValue(".*")
```

Comment:
`noDefaultValue` should be more suitable.

Reply:
> `noDefaultValue` should be more suitable.

Yes! I have updated it~